homeworks/serhii.ripchanskyi_Ripchanskiy/11-dependency-injection/vechicle/src/app/app.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component } from '@angular/core';
+import { finalize } from 'rxjs/operators';
+
+import { Vehicle } from './models/';
+import { VehicleService } from './services';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss']
+})
+export class AppComponent {
+  title = 'vehicle';
+  vehicles: Vehicle[] = [];
+  areVehiclesLoading: boolean = false;
```

Comment:
I suggest you set default value as `true`. When you open the page, list will be loading, so the default status is `true`